training_record/serializers.py

```diff
@@ -55,6 +70,7 @@ class RecordSerializer(serializers.ModelSerializer):
     off_campus_event = OffCampusEventSerializer(read_only=True)
     attachments = RecordAttachmentSerializer(many=True, read_only=True)
     contents = RecordContentSerializer(many=True, read_only=True)
+    feedback = CampusEventFeedbackSerializer(many=True, read_only=True)
```

Comment:
As I commented in #60, please do not add more nested serializer. Instead, retrieve instances independently.